Dockerfile

```diff
@@ -1,4 +1,19 @@
-FROM alpine:3.11
+#Add qemu function for supporting multiarch
+ARG IMAGEARCH
+FROM alpine:3.12 as qemu
+RUN apk add --no-cache curl
+ARG QEMUVERSION=4.0.0
+ARG QEMUARCH
+
+SHELL ["/bin/ash", "-o", "pipefail", "-c"]
```

Comment:
🙋 Is it necessary to change the default shell options? You could break the `curl | tar` command into separate `RUN` steps and the resulting build would be easier to debug (e.g. if there were issues with a network connection to https://github.com).